### PR TITLE
Additions and corrections to contribute docs

### PR DESCRIPTION
**Pull request by ignotus666**
# Does this need translation?

- [ ] Yes 
- [*] No 

# Changes

Corrected "adminstration" -> "administration".
Corrected spaces before units and some other errors
Added note about abbreviations

### CODE REVIEW

**Comment by ignotus666** [comment]
I wanted to make a separate PR for the corrections to wiki/en docs against 'release', as I will also update the .po files to avoid turning those paragraphs into fuzzies. But I'm unsure whether this would create a conflict with #533 so I'll wait until that gets merged or discarded.

File: contribute/en/Administration.md
Comment (pljones) [inline contribute/en/Administration.md:124]: No hyphen.

Comment (ignotus666) [reply contribute/en/Administration.md:124]: There are differing views on that but I was following the usage [here](https://dictionary.cambridge.org/dictionary/english/decision-making), [here](https://www.collinsdictionary.com/dictionary/english/decision-making) and [here](https://www.merriam-webster.com/dictionary/decision-making), where the noun form uses a hyphen.

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
Why doesn’t this go to release?

**Comment by ignotus666** [comment]
Good question. For some reason I was convinced these documents had been added to next-release but didn't exist yet in release... I'll make a new PR.

File: contribute/en/Administration.md
Comment (pljones) [reply contribute/en/Administration.md:124]: Adjectival I'd agree with (_decision-making process_; the only example given in Collins, which is poor...) but straight noun form it makes me feel it's wrong...  Cambridge seems much more set on the hyphen...  Apparently it's because I'm old fashioned.  Which I guess is now "old-fashioned"...

**Pull request closed without merging**
